frontend/pages/views/PhotographerListView.js

```diff
@@ -192,78 +260,123 @@ export class PhotographerListView extends React.Component {
+                                        }}
+                                    />
+                                    <DropDown
+                                        id="alph-filter"
+                                        blue={true}
+                                        items={this.state.dropdownSearchOptions.nameStartsWith}
+                                        placeholder={"Alphabet"}
+                                        activeDropdown={this.state.activeDropdown}
+                                        toggleActiveDropdown={this.toggleActiveDropdown}
+                                        value={this.state.searchQueries.nameStartsWith}
+                                        onChange={(value) => {
+                                            this.setState((prev) => ({
+                                                searchQueries: {
+                                                    ...prev.searchQueries,
+                                                    nameStartsWith: value,
+                                                },
+                                            }));
+                                        }}
                                     />
                                 </div>
                             </div>
-                        </form>
-                    </div>
+                            <div className="sortBy-container">
+                                <p>Sort by:</p>
+                                <DropDown
+                                    id="sort"
+                                    blue={false}
+                                    items={this.state.dropdownSearchOptions.orderBy}
+                                    placeholder={"---"}
+                                    activeDropdown={this.state.activeDropdown}
+                                    toggleActiveDropdown={this.toggleActiveDropdown}
+                                    value={this.state.sortBy}
+                                    onChange={(value) => {
+                                        this.setState({
+                                            sortBy: value,
+                                        });
+                                    }}
+                                />
+                            </div>
+                        </div>
+                    </form>
+                </div>
 
-                    <div className="photographerGallery">
-                        <ul className="list-inline">{this.getPhotoList()}</ul>
-                    </div>
+                <div className="photographerGallery">
+                    <ul className="list-inline" onScroll={this.handleScroll}>
+                        {this.getPhotoList()}
+                        <div className="photographers-results-footer">
+							{this.state.isLastPage
+								? (this.state.photographers.length === 0
+									? "No photographers found that match your search query!"
+									: "End of Results!!!")
```

Comment:
Generally, don't leave temporary copy like this in the frontend. You want it to look professional... especially if you forget to change it later!